core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -338,6 +344,44 @@ public static Buffer getPayload(final Message msg) {
         return null;
     }
 
+    /**
+     * Gets a message's body as Buffer object as {@link String}.
```

Comment:
this seems to be garbled
also, there is no need to link to the String class, is there?